modules/videopress/utility-functions.php

```diff
@@ -1,99 +1,6 @@
 <?php
-
 /**
- * Validate user-supplied guid values against expected inputs
+ * Deprecated. Functions moved to functions.compat.php
```

Comment:
Oops, my bad. I removed "deprecated" in d7850ac60e9306126eb5f2aa2a97b0bbde247a0b

Also worth noting that I added some caching to the `videopress_get_video_details` function in 29a4ba25c54436900b60e7ef6f11fa4282416385, as per https://github.com/Automattic/jetpack/pull/3853#discussion_r64011347